packages/is-shallow-equal/test/index.js

```diff
@@ -46,9 +46,9 @@ describe( 'isShallowEqual', () => {
 		expect( isShallowEqual( b, a ) ).toBe( false );
 	} );
 
-	it( 'returns false if b object has different key than a', () => {
```

Comment:
The diff makes it look like this was changed, but in fact I had removed this test case altogether as considered redundant with the previous case, which calls `isShallowEqual` with the objects in _both_ possible arguments order arrangements.

See 100ab09